src/pages/Login/ResetPassword.js

```diff
@@ -1,38 +1,82 @@
 
+/*
+    Password doesn't match hidden is false.
 
+    submits if password does not matches it became true
+*/
 import React, { Component } from 'react'
-import './Login.css';
 import Form from './Form';
 import './ResetPassword.css';
 
- class ResetPassword extends Component {
-     
+class ResetPassword extends Component {
+    constructor(props){
+        super(props);
+        this.state = {
+            password: "",
+            checkPassword: "",
+            passwordMatch: false,
+        };
+    }
+
+    handleSubmit = event => {
+        event.preventDefault();
+       
+        if(this.state.password !== this.state.checkPassword){
+            this.setState({passwordMatch: true});
+        }else{
+            this.setState({passwordMatch: false});
+        }
+
+       // Clear inputs. Inputs won't clear for some reason.
+       	this.setState({password: ""})
+		this.setState({checkPassword: ""});
+        console.log(this.state.password);
+        console.log(this.state.checkPassword);
+ 
```

Comment:
`// When logging inside the callback from setState, it will show the correctly updated value
        
       this.setState({password:''}, () => {
           console.log(this.state.password);
       })
       	this.setState({checkPassword:''}, ()=> {
               console.log(this.state.checkPassword);
           });`